canonical-web-and-design/vanilla-framework
# Rename 'individual' examples to 'standalone'

## Done

Rename 'individual' example files to 'standalone' (in code and URLs).

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/ or [demo](https://vanilla-framework-canonical-web-and-design-pr-2776.run.demo.haus/)
- Go to standalone examples https://vanilla-framework-canonical-web-and-design-pr-2776.run.demo.haus/examples/standalone
- Check any examples, they should work as before
